app/serializers/search/user_serializer.rb

```diff
@@ -1,5 +1,8 @@
 module Search
   class UserSerializer < ApplicationSerializer
+    CUSTOM_ATTRIBUTES = "custom_attributes".freeze
+    HASH_TRANSFORM = ->(key, value) { { name: key, value: value } }
```

Comment:
I needed the exact same transformation in two places and a lambda was convenient for this purpose. If we end up needing this elsewhere I will move it into a refinement or something.